.github/workflows/node.js.yml

```diff
@@ -10,31 +7,48 @@ on:
   pull_request:
 
 jobs:
-  build:
+  test:
+    name: Run tests on supported Node.js versions
```

Comment:
I've added name labels to all the jobs and steps in the workflow, to make it more explicit what they do and easier to read back.